CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -804,9 +810,198 @@ private static void OnTimeoutEvent(object source, ElapsedEventArgs e)
             timer.Stop();
             timer.Dispose();
         }
+        public class PotManager
+        {
+            // List to hold the pots for the game
+            public List<Pot> Pots;
+
+            public int Total { get; set; }
+            public PotType PotType { get; set; }
+            public List<int> EligiblePlayers { get; set; }
+
+            public PotManager()
+            {
+                Pots = new List<Pot>(); // Initialize the Pots list
+            }
+            /// <summary>
+            /// Manages the pot in a Texas Hold'em game, including adding and distributing chips.
+            /// </summary>
+            public class Pot
+            {
+                public int Total { get; set; }
+                public PotType PotType { get; set; }
+                public List<int> EligiblePlayers { get; set; }
 
+                public Pot(PotType type, int total, List<int> eligiblePlayers)
+                {
+                    PotType = type;
+                    Total = total;
+                    EligiblePlayers = eligiblePlayers;
+                }
+            }
+            /// <summary>
+            /// Initialize the main pot, if any players are short side pots will be created.
+            /// </summary>
+            /// <param name="_ante">The ante needed to be added by each player to the pot.</param>
+            /// <param name="playerBets">List of antes values to be added to the pot.</param>
+            /// /// <param name="playersActive">Importing a list of all player indecies, all players eligible for main pot</param>
+            public void InitializePot(int _ante, List<int> playerBets, List<int> playersActive)
+            {
+                Pots.Add(new Pot(PotType.MAIN, 0, playersActive));
+
+                if (playerBets.All(n => n == playerBets[0])) //if all bets match the ante
+                {
+                    AddToPot(_ante, playerBets, playersActive);
+                }
+                else //create side pots if any bets do not match the ante
+                {
+                    CreateSidePots(_ante, playerBets, playersActive);
+                }
+            }
+            /// <summary>
+            /// Adds a bets to the pot.
+            /// </summary>
+            /// <param name="_currentBet">The amount each player is contributing to the pot.</param>
+            /// <param name="numPlayers">The number of players contributing to the pot.</param>
```

Comment:
make sure the summary comments remain active. Currently, this is referencing a parameter that is no longer there (`numPlayers`).